Venus-venuslangmuir/Homework-1/BONUS-random-address-generator/app.js

```diff
@@ -0,0 +1,36 @@
+/*
+Optional bonus work on conditionals and functions:
+
+    Write a script that can generate random addresses
+    As a first step, create arrays that contain dummy data for each of the following: street number, street name, city name, state name, and zip code
+    Your script should randomly select one item from each of these arrays and then use them to construct a random address
+    Each time you load or reload the page in a browser, your program should print a new randomly-generated address to the terminal. For example:
+    
+    34578 Dolphin Street, Wonka NY, 44506
+*/
+
+// Arrays of data
+let streetNumber = [1234, 5678, 9101, 1121, 3141, 5161, 7181, 9202, 1222, 3242];
+let streetName = ['Nowhere', 'Rosemary', 'Catnip', 'Somewhere', 'Lily', 'Willow', 'Sleepy', 'Seventeen', 'Camper', 'General'];
+let cityName = ['Olympia', 'Charleston', 'Raleigh', 'Concord', 'Helena', 'Salem', 'Albuquerque', 'Milwaukee', 'Fargo', 'San Francisco'];
+let stateName = ['KS', 'OK', 'MD', 'AR', 'CA', 'PA', 'CT', 'DE', 'FL', 'GA' ];
+let zipCode = [90001, 90002, 90003, 90004, 90005, 90006, 90007, 90008, 90009, 90010];
+
+
+// Generate 5 different random number to make it truly random combination of fake address
+let randomNumber1 = Math.floor(Math.random() * streetNumber.length);
```

Comment:
It often helps to name the variables something that infers what the value will be - so when you use it later on you dont wonder if randomNumber1 is correctly assigned to the right array.